### PR TITLE
Added welcome modal

### PR DESCRIPTION
**Pull request by joecohens**
Show a welcome modal instead of the steps to configure. https://github.com/cachethq/Cachet/pull/362 and https://github.com/cachethq/Cachet/issues/99 

screen shot 2015-01-18 at 3 57 48 pm

### CODE REVIEW

File: app/lang/de/dashboard.php [hunk 1]
```diff
@@ -128,4 +128,19 @@
         'whoops'        => 'Oops.',
     ],
 
+    // Welcome modal
+    'welcome' => [
+        'welcome' => 'Welcome to Cachet',
+        'message' => 'Your status page is almost ready! You might want to configure these extra settings',
+        'close'   => 'Just go straight to my dashboard',
+        'steps'   => [
+            'component'   => 'Create components',
```
Comment (GrahamCampbell) [inline app/lang/de/dashboard.php]: 1 too many spaces in this aligment

File: app/lang/en/dashboard.php [hunk 2]
```diff
@@ -135,4 +135,19 @@
         'whoops'        => 'Whoops.',
     ],
 
+    // Welcome modal
+    'welcome' => [
+        'welcome' => 'Welcome to Cachet',
+        'message' => 'Your status page is almost ready! You might want to configure these extra settings',
+        'close'   => 'Just go straight to my dashboard',
+        'steps'   => [
+            'component'   => 'Create components',
```
Comment (GrahamCampbell) [inline app/lang/en/dashboard.php]: ...

File: app/lang/fr/dashboard.php [hunk 3]
```diff
@@ -134,4 +134,19 @@
         'whoops'        => 'Oups.',
     ],
 
+    // Welcome modal
+    'welcome' => [
+        'welcome' => 'Welcome to Cachet',
+        'message' => 'Your status page is almost ready! You might want to configure these extra settings',
+        'close'   => 'Just go straight to my dashboard',
+        'steps'   => [
+            'component'   => 'Create components',
```
Comment (GrahamCampbell) [inline app/lang/fr/dashboard.php]: ...

File: app/lang/de/dashboard.php (hunk 1 above)
Comment (joecohens) [inline app/lang/de/dashboard.php]: Fixed :)

**Pull request merged**